pyproject.toml

```diff
@@ -126,12 +127,15 @@ allow_redefinition = true
 namespace_packages = true
 show_error_codes = true
 plugins = ["pydantic.mypy"]
-exclude = ["tests/*"]
+# We use reflection a lot in this codebace, and mypy does not understand that
+# finding all subclasses of an abstract class gives you concrete implementations.
+# Thus, we disable the type-abstract error.
+disable_error_code = ["type-abstract"]
 
 [tool.pydantic-mypy]
 init_forbid_extra = true
 init_typed = true
-warn_required_dynamic_aliases = true
+warn_required_dynamic_aliases = false
```

Comment:
In the new pydantic classes we want to use dynamic alias. This ensures that all alias are camel case, instead of manually setting it.